tibiawikisql/utils/charms.py

```diff
@@ -0,0 +1,102 @@
+import json
+import os
+from contextlib import closing
+
+import pkg_resources
+import time
+
+import requests
+
+from tibiawikisql.utils import ENDPOINT, headers
+
+
+def save_charm_list(con):
+    print("Parsing charm information from json...")
+    start_time = time.time()
+    with open(pkg_resources.resource_filename(__name__, "charms.json")) as file:
+        charms = json.load(file)
+    c = con.cursor()
+    for charm in charms:
+        c.execute("INSERT INTO charm(name, type, description, points) VALUES(?,?,?,?)",
+                  (charm["name"], charm["type"], charm["description"], charm["points"]))
```

Comment:
Seems like you are adding `"100 points"` to the `points` column, which is declared as a `INTEGER` type.